src/main/kotlin/it/pagopa/wallet/config/WalletMigrationConfig.kt

```diff
@@ -0,0 +1,6 @@
+package it.pagopa.wallet.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+
+@ConfigurationProperties(prefix = "wallet.migration")
+data class WalletMigrationConfig(val cardPaymentMethodId: String)
```

Comment:
How come you created a configuration ConfigurationProperties class ?
From what I can see you currently have only one environment variable wallet.migration.cardPaymentMethodId, do you provide others with the same prefix wallet.migration ? 🙂

Reply:
Actually there aren't others variables with same prefix. The creation of an explicit configuration is to get a more "clean" injection here, and because I prefer to inject @Value directly to objects annotated with @Bean.
```kotlin
@Service
class MigrationService(
    ....
    private val uniqueIdUtils: UniqueIdUtils,
    walletMigrationConfig: WalletMigrationConfig
) {
```
Instead of:
```kotlin
@Service
class MigrationService(
    ....
    private val uniqueIdUtils: UniqueIdUtils,
    @Value("\${wallet.migration.cardPaymentMethodId}") cardPaymentMethodId: String
) {
```
 But for just one we can also inject it directly, what do you think?

Reply:
If you feel that over time you may need to have variables with the same prefix then di say leave it that way. Otherwise I personally would use the @Value

Reply:
We will probably have other configurations to add to the migration process, so dedicated configuration sounds good